packages/block-editor/src/hooks/color-panel.js

```diff
@@ -2,29 +2,31 @@
  * WordPress dependencies
  */
 import { useState, useEffect } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import PanelColorGradientSettings from '../components/colors-gradients/panel-color-gradient-settings';
 import ContrastChecker from '../components/contrast-checker';
-import InspectorControls from '../components/inspector-controls';
 import { __unstableUseBlockRef as useBlockRef } from '../components/block-list/use-block-props/use-block-refs';
+import ColorGradientControl from '../components/colors-gradients/control';
+import useCommonSingleMultipleSelects from '../components/colors-gradients/use-common-single-multiple-selects';
+import useSetting from '../components/use-setting';
 
 function getComputedStyle( node ) {
 	return node.ownerDocument.defaultView.getComputedStyle( node );
 }
 
 export default function ColorPanel( {
-	settings,
+	setting,
 	clientId,
 	enableContrastChecking = true,
-	showTitle = true,
 } ) {
 	const [ detectedBackgroundColor, setDetectedBackgroundColor ] = useState();
 	const [ detectedColor, setDetectedColor ] = useState();
 	const ref = useBlockRef( clientId );
+	const colorGradientSettings = useCommonSingleMultipleSelects();
+	colorGradientSettings.colors = useSetting( 'color.palette' );
```

Comment:
We were showing multiple color palettes default, theme, and user, and now we are just showing a single palette.